applications/NXsas.nxdl.xml

```diff
@@ -133,40 +130,34 @@
         <field name="y_pixel_size" type="NX_FLOAT" units="NX_LENGTH">
           <doc>Size of a pixel in y direction</doc>
         </field>
-        <field name="polar_angle" type="NX_FLOAT" units="NX_ANGLE"></field>
-        <field name="azimuthal_angle" type="NX_FLOAT"
-          units="NX_ANGLE">
-        </field>
-        <field name="rotation_angle" type="NX_FLOAT" units="NX_ANGLE"></field>
-        <field name="aequatorial_angle" type="NX_FLOAT"
-          units="NX_ANGLE">
+        <field name="polar_angle" type="NX_FLOAT" units="NX_ANGLE" minOccurs="0" />
+        <field name="azimuthal_angle" type="NX_FLOAT" units="NX_ANGLE" minOccurs="0" />
+        <field name="rotation_angle" type="NX_FLOAT" units="NX_ANGLE" minOccurs="0" />
+        <field name="aequatorial_angle" type="NX_FLOAT" units="NX_ANGLE" minOccurs="0" />
+        <field name="beam_center_x" type="NX_FLOAT" units="NX_LENGTH">
```

Comment:
Should this be optional as the doc for `data` says it's not necessary. Also, shape of that `data` field is column-major - is that correct?

Reply:
The doc for `data` should be adjusted.

Reply:
doc for beam center should indicate that this is determined in data reduction

Reply:
We'll accept that the ordering directions (the `dimensions`) as presented should remain.  At least until further review from the community.

Reply:
It's necessary to specify which is x and y to coordinate with the pixel dimensions, so as to compute angle and then Q.

Reply:
@PeterC-DLS By your suggestion, it should be possible to replace both the symbol table and the specification of array dimensions by use of the `SIGNAL_indices` attribute, this allowing the use of additional parameters in the `data` array such a time, temperature, electric field, ...

Reply:
After discussion at NIAC2022, it is not necessary to refactor with `NXdata@SIGNAL_indices` (this use is possible, via NXdata).  Will not change.